File: src/main/java/com/ject/studytrip/mission/domain/repository/DailyMissionRepository.java
package com.ject.studytrip.mission.domain.repository;

import com.ject.studytrip.mission.domain.model.DailyMission;
import java.util.List;

public interface DailyMissionRepository {
    DailyMission save(DailyMission dailyMission);

    List<DailyMission> saveAll(List<DailyMission> dailyMissions);

    List<DailyMission> findAllByIdIn(List<Long> ids);
}

File: src/main/kotlin/com/ject/studytrip/mission/domain/repository/DailyMissionRepository.kt
package com.ject.studytrip.mission.domain.repository

import com.ject.studytrip.mission.domain.model.DailyMission

interface DailyMissionRepository {
    fun save(dailyMission: DailyMission): DailyMission

    fun saveAll(dailyMissions: List<DailyMission>): List<DailyMission>

    fun findAllByIdIn(dailyMissionIds: List<Long>): List<DailyMission>
}
